string summary
string group
string planner
string result
float64 planning_time
int32 trajectory_size
float64 trajectory_duration
int32 state_allocator_size
